src/coredump/coredump.c

```diff
@@ -819,8 +823,7 @@ static int submit_coredump(
         _cleanup_close_ int coredump_fd = -EBADF, coredump_node_fd = -EBADF;
         _cleanup_free_ char *filename = NULL, *coredump_data = NULL;
         _cleanup_free_ char *stacktrace = NULL;
-        _cleanup_free_ char *root = NULL;
-        const char *module_name;
+        const char *module_name, *root = MOUNT_TREE_ROOT;
```

Comment:
this is confusing to me? now you pass this along always, even for coredumps of the host rather than any container? that cannot be right? what am i missing?

Reply:
In `parse_core()` we check root argument with `empty_or_root()` and don't call `sym_dwfl_set_sysroot()` is such case.